suricata/update/data/index.py

```diff
@@ -1 +1 @@
description': 'Broad ruleset composed of malware rules and other security-related countermeasures, and curated by the Secureworks Counter Threat Unit research team.\n', 'vendor': 'Secureworks', 'license': 'Commercial', 'url': 'https://ws.secureworks.com/ti/ruleset/%(secret-code)s/Suricata_suricata-security_latest.tgz', 'parameters': {'secret-code': {'prompt': 'Secureworks Threat Intelligence Authentication Token'}}, 'subscribe-url': 'https://www.secureworks.com/contact/ (Please reference CTU Countermeasures)', 'min-version': '3.0.0'}, 'sslbl/ssl-fp-blacklist': {'summary': 'Abuse.ch SSL Blacklist', 'description': 'The SSL Blacklist (SSLBL) is a project of abuse.ch with the goal of detecting malicious SSL connections, by identifying and blacklisting SSL certificates used by botnet C&C servers. In addition, SSLBL identifies JA3 fingerprints that helps you to detect & block malware botnet C&C communication on the TCP layer.\n', 'vendor': 'Abuse.ch', 'license': 'Non-Commercial', 'url': 'https://sslbl.abuse.ch/blacklist/sslblacklist.rules', 'checksum': False}, 'sslbl/ja3-fingerprints': {'summary': 'Abuse.ch Suricata JA3 Fingerprint Ruleset', 'description': "If you are running Suricata, you can use the SSLBL's Suricata JA3 FingerprintRuleset to detect and/or block malicious SSL connections in your network based on the JA3 fingerprint. Please note that your need Suricata 4.1.0 or newer in order to use the JA3 fingerprint ruleset.\n", 'vendor': 'Abuse.ch', 'license': 'Non-Commercial', 'url': 'https://sslbl.abuse.ch/blacklist/ja3_fingerprints.rules', 'min-version': '4.1.0', 'checksum': False}, 'etnetera/aggressive': {'summary': 'Etnetera aggressive IP blacklist', 'vendor': 'Etnetera a.s.', 'license': 'MIT', 'url': 'https://security.etnetera.cz/feeds/etn_aggressive.rules', 'min-version': '4.0.0', 'checksum': False}, 'tgreen/hunting': {'summary': 'Threat hunting rules', 'description': 'Heuristic ruleset for hunting. Focus on anomaly detection and showcasing latest engine features, not performance.\n', 'vendor': 'tgreen', 'license': 'GPLv3', 'url': 'https://raw.githubusercontent.com/travisbgreen/hunting-rules/master/hunting.rules', 'min-version': '4.1.0', 'checksum': False}, 'malsilo/win-malware': {'summary': 'Commodity malware rules', 'description': 'TCP/UDP, DNS and HTTP Windows threats artifacts observed at runtime.\n', 'vendor': 'malsilo', 'license': 'MIT', 'url': 'https://malsilo.gitlab.io/feeds/dumps/malsilo.rules.tar.gz', 'min-version': '4.1.0', 'homepage': 'https://raw-data.gitlab.io/post/malsilo_2.1/', 'checksum': True}, 'stamus/lateral': {'summary': 'Lateral movement rules', 'description': 'Suricata ruleset specifically focused on detecting lateral\nmovement in Microsoft Windows environments by Stamus Networks\n', 'vendor': 'Stamus Networks', 'min-version': '6.0.6', 'license': 'GPL-3.0-only', 'support-url': 'https://discord.com/channels/911231224448712714/911238451842666546', 'url': 'https://ti.stamus-networks.io/open/stamus-lateral-rules.tar.gz'}}}
```

Comment:
it would be a lot nicer to have this expanded so it can actually be sanely reviewed...

Reply:
See: https://github.com/OISF/suricata-update/pull/325